### ISSUE TITLE
h1 Voiceover implementation on OD TV 

### ISSUE THREAD

**Issue by rebeccamcginn** [bbc/simorgh#7062]
**Describe the bug**

```<!-- -->``` is being output in the h1 in the DOM, I believe this is causing text splitting with VoiceOver on iOS - This should be removed so the h1 is announced with one swipe and so that 'comma' is not announced - this should just represent a pause with voiceover.

This to do with how strings are interpolated in JSX. e.g. you can do

```${offscreenText}, ${moreOffscreenText}```

instead of

```
offscreenText
','
moreOffscreenText
```

**To Reproduce**
Steps to reproduce the behaviour:

1. Go to https://www.test.bbc.com/somali/bbc_somali_tv/tv_programmes/w13xttqt
2. Inspect the h1
3. Expect to see "Caawa Iyo Caalamka, 2 Luulyo 2020" without JSX comments

**Expected behaviour**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Testing notes**

- [ ] This bug fix is expected to need manual testing.

**Additional context**

**Comment by LilyL0u** [comment]
No  in DOM. Catharine checked voiceover. I checked talkback, JAWS and NVDA and date seems to be read correctly (splitting of the date).
